sw/device/tests/sim_dv/sram_ctrl_execution_main_test.c

```diff
@@ -212,10 +212,11 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_lc_ctrl_init(
       mmio_region_from_addr(TOP_EARLGREY_LC_CTRL_BASE_ADDR), &lc));
 
+  bool debug_func;
```

Comment:
Nit: good idea to initialize the variable in the declaration, see [here](https://google.github.io/styleguide/cppguide.html#Local_Variables), since depending on what happens in `lc_ctrl_testutils_debug_func_enabled()`, the variable may not get set.